psutil/net.go

```diff
@@ -62,6 +62,17 @@ var netIOCounterLabels = map[string]label{
 	},
 }
 
+var netConntrackCounterLabels = map[string]label{
```

Comment:
Metric returned by GetMetricTypes is in static list (netIOCounterLabels and netConntrackCounterLabels), and in case of netfilter is not loaded in kernel, task with this metric will fail and plugin crash. In the reason why i separate "basic" net metric and netfilter metric.